pywr/parameters/parameters.py

```diff
@@ -139,8 +139,20 @@ def _value_to_interpolate(self, ts, scenario_index):
     @classmethod
     def load(cls, model, data):
         node = model._get_node_from_ref(model, data.pop("node"))
-        volumes = np.array(data.pop("volumes"))
-        values = np.array(data.pop("values"))
+        volumes = data.pop("volumes")
+        if isinstance(volumes, list):
+            volumes = np.array("volumes")
```

Comment:
This should be:

```
volumes = np.array(volumes)
```

Reply:
fixed !

Reply:
however, the pytest tests still isn't able to read the .csv input as an array, as the problem in the test appears to be linked to the interpolation itself (it doesn't recieve an array, or perhaps not one it can read). 
Could you confirm this?